Subject:  Fix breakage with CEF 93.1.11 (cr-93.0.4577.63), NULL -> nullptr, 0 -> nullptr
Patch Author:  Orson Teodoro <orsonteodoro@hotmail.com>
Date:  Sun Sep  5 10:22:04 AM PDT 2021 (Unix time: 1630862524)

This applies to obs-browser f1a61c5a2579e5673765c31a47c2053d4b502d4b

diff -urp obs-browser-f1a61c5a2579e5673765c31a47c2053d4b502d4b.orig/browser-app.cpp obs-browser-f1a61c5a2579e5673765c31a47c2053d4b502d4b/browser-app.cpp
--- obs-browser-f1a61c5a2579e5673765c31a47c2053d4b502d4b.orig/browser-app.cpp	2021-05-30 10:51:38.000000000 -0700
+++ obs-browser-f1a61c5a2579e5673765c31a47c2053d4b502d4b/browser-app.cpp	2021-09-05 10:18:28.742547094 -0700
@@ -110,7 +110,7 @@ void BrowserApp::OnContextCreated(CefRef
 {
 	CefRefPtr<CefV8Value> globalObj = context->GetGlobal();
 
-	CefRefPtr<CefV8Value> obsStudioObj = CefV8Value::CreateObject(0, 0);
+	CefRefPtr<CefV8Value> obsStudioObj = CefV8Value::CreateObject(nullptr, nullptr);
 	globalObj->SetValue("obsstudio", obsStudioObj,
 			    V8_PROPERTY_ATTRIBUTE_NONE);
 
@@ -156,7 +156,7 @@ void BrowserApp::ExecuteJSFunction(CefRe
 	CefRefPtr<CefV8Value> jsFunction = obsStudioObj->GetValue(functionName);
 
 	if (jsFunction && jsFunction->IsFunction())
-		jsFunction->ExecuteFunction(NULL, arguments);
+		jsFunction->ExecuteFunction(nullptr, arguments);
 
 	context->Exit();
 }
@@ -312,7 +312,7 @@ bool BrowserApp::OnProcessMessageReceive
 
 		CefRefPtr<CefV8Value> dispatchEvent =
 			globalObj->GetValue("dispatchEvent");
-		dispatchEvent->ExecuteFunction(NULL, arguments);
+		dispatchEvent->ExecuteFunction(nullptr, arguments);
 
 		context->Exit();
 
@@ -342,7 +342,7 @@ bool BrowserApp::OnProcessMessageReceive
 		args.push_back(retval);
 
 		if (callback)
-			callback->ExecuteFunction(NULL, args);
+			callback->ExecuteFunction(nullptr, args);
 
 		context->Exit();
 
